File: Chapter-3/Java/Problem2.java
import java.util.Scanner;

public class Problem2 {
    public static void main(String[] args) {
        int ram, shyam, ajay;
        Scanner sc = new Scanner(System.in);
        ram = sc.nextInt();
        shyam = sc.nextInt();
        ajay = sc.nextInt();

        if (ram < shyam && ram < ajay) {
            System.out.println("Ram is youngest");
        } else if (shyam < ram && shyam < ajay) {
            System.out.println("Shyam is youngest");
        } else {
            System.out.println("Ajay is youngest");
        }
        sc.close();
    }
}

File: Chapter-3/Python/Problem2.py
ram,shyam,ajay = list(map(int,input().split()))
if ram < shyam and ram < ajay:
    print("Ram is youngest")
elif shyam < ram and shyam < ajay:
    print("Shyam is youngest")
else:
    print("Ajay is youngest")